documentation/IDTA-01005/modules/ROOT/pages/IDTA-01005_Bibliography.adoc

```diff
@@ -36,16 +36,16 @@ SPDX-License-Identifier: CC-BY-4.0
 [8] T. Preston-Werner "Semantic Versioning". Version 2.0.0. Accessed: 2020-11-13. [Online]. Available: https://semver.org/spec/v2.0.0.html
 
 [#bib9]
-[9] "Details of the Asset Administration Shell – The exchange of information between partners in the value chain of Industrie 4.0". [Online]. Available: https://industrialdigitaltwin.org/en/content-hub
+[9] IDTA-01001 "Specification of the Asset Administration Shell. Part 1: Metamodel". Industrial Digital Twin Assocation. [Online]. Available: https://industrialdigitaltwin.org/en/content-hub
```

Comment:
IDTA-01005 should work with any version of the metamodel. We do not know in which release it will be contained in future. So I prefer to not update the link to a concrete version but to https://industrialdigitaltwin.org/en/content-hub/aasspecifications only